package.json

```diff
@@ -39,5 +39,6 @@
   "dependencies": {
     "@babel/runtime": "^7.11.2",
     "@teamawesome/access": "^1.0.7"
-  }
+  },
+  "types": "./multiDict.d.ts"
```

Comment:
Could you rename this to  `./multi-dict.d.ts` to keep the name the same as the package?